examples/pubsub/README.md

```diff
@@ -76,6 +78,6 @@ dapr stop --app-id sub
 
 ```shell
 == APP == 2023/03/29 21:36:07 event - PubsubName: messages, Topic: neworder, ID: 82427280-1c18-4fab-b901-c7e68d295d31, Data: ping
-== APP == 2023/03/29 21:36:07 event - PubsubName: messages, Topic: neworder, ID: cc13829c-af77-4303-a4d7-55cdc0b0fa7d, Data: multi-pong
-== APP == 2023/03/29 21:36:07 event - PubsubName: messages, Topic: neworder, ID: 0147f10a-d6c3-4b16-ad5a-6776956757dd, Data: multi-ping
+== APP == 2023/03/29 21:36:07 event - PubsubName: messages, Topic: newbulkorder, ID: cc13829c-af77-4303-a4d7-55cdc0b0fa7d, Data: multi-pong
+== APP == 2023/03/29 21:36:07 event - PubsubName: messages, Topic: newbulkorder, ID: 0147f10a-d6c3-4b16-ad5a-6776956757dd, Data: multi-ping
```

Comment:
Why is this replacing an existing test for the standard pubsub queue?

Reply:
Since I added bulksubscribe so It would generate that events too right ? So I had to add it

Reply:
I am making sure bulksubscribe listens to publishEvents  and normal subscribe listens to publishEvent

Reply:
Existing pub/sub examples should be kept so far as is possible. You've removed the non-bulk subscription validation by this.

Reply:
first one is non-bulk and other 2 are bulk .